scopes/scope/sign/sign.cmd.ts

```diff
@@ -2,47 +2,57 @@ import chalk from 'chalk';
 import { Command, CommandOptions } from '@teambit/cli';
```

Comment:
these changes are from another PR that's already merged to master... Due to overzealous re-basing on my part